pages/test-evaluate/evaluation-tools/selecting-eval-tools.pl.md

```diff
@@ -0,0 +1,229 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:".
+title: Selecting Web Accessibility Evaluation Tools
+lang: pl  # Change "en" to the translated-language shortcode
+last_updated: 2024-06-27  # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+translators: # remove from the beginning of this line and the lines below: "# " (the hash sign and the space)
+- name: "Stefan Wajda" # Add one -name: line for every translator
+# contributors:
+# - name: "Jan Doe"   # Replace Jan Doe with name, or delete this line if not multiple contributors
+
+github:
+  label: wai-selecting-eval-tools
+
+permalink: /test-evaluate/tools/selecting/pl  # Add the language shortcode to the end, with no slash at end, for example: /link/to/page/fr
+ref: /test-evaluate/tools/selecting/  # Do not change this
+
+# In the footer below:
+# Do not change the dates
+# Translate the other words below, including "Date:" and "Latest editors and developers:"
+# Translate the Working Group and projects names. Leave the Working Group and projects acronyms in English.
+footer: >
+   <p><strong>Data:</strong> Informacje o konkretnych narzędziach są często aktualizowane, gdy je otrzymujemy. W informacjach dla każdego narzędzia znajduje się data ostatniej aktualizacji. Interfejs użytkownika Lista narzędzi oceny został odnowiony w grudniu 2023 r. Po raz pierwszy opublikowany w marcu 2006 roku.</p>
+   <p><strong>Najnowsi redaktorzy i programiści:</strong> <a href="https://www.w3.org/People/kevin/">Kevin White</a>, Eric Velleman, Michel Hansma i Vera Lange. <strong>Opiekun treści:</strong> José Ramón Hilera González. <a href="https://www.w3.org/WAI/ER/tools/acknowledgements">Podziękowania zawierają listę</a> współpracowników i poprzednich redaktorów, w tym <a href="https://www.w3.org/groups/wg/eowg/participants/">uczestników EOWG, </a>Erica Eggerta i Shadi Abou-Zahra.</p>
+   <p>Opracowane przez Grupę Roboczą ds. Edukacji i Promocji (<a href="https://www.w3.org/WAI/EO/">EOWG</a>) przy wsparciu <a href="https://www.w3.org/WAI/TIES/">Projektu WAI-TIES</a> w 2006, zaktualizowane przy wsparciu <a href="https://www.w3.org/WAI/ACT/">Projektu WAI-ACT</a> w 2014, i zmienione w 2023 roku w ramach <a href="https://www.w3.org/WAI/about/projects/wai-coop/">Projektu WAI-CooP</a>, współfinansowanego przez Komisję Europejską. </p>  
+   
+---
+
+{::nomarkdown}
+{% include box.html type="start” h="2" title="Wprowadzenie” class="full”%}
```

Comment:
@zwiastunsw The build has failed. This is probably due to the missing space before closing the include, and/or due to the different quotation marks used.

Reply:
@remibetin thx, corrected